src/emc/task/emccanon.cc

```diff
@@ -2599,6 +2598,12 @@ void INIT_CANON()
 	    ("non-standard length units, setting interpreter to mm");
 	canon.lengthUnits = CANON_UNITS_MM;
     }
+    /* Set blending tolerance default depending on units machine is based on*/
+    if (canon.lengthUnits == CANON_UNITS_INCHES) {
+        SET_MOTION_CONTROL_MODE(CANON_CONTINUOUS, .001);
+    } else {
+        SET_MOTION_CONTROL_MODE(CANON_CONTINUOUS, .0254);
```

Comment:
Perhaps the code intent is expressed more clearly if you use something like .001 * MM_PER_INCH instead of .0254?  The compiler will end up with the same value, so there will be no speed impact, but it will be more obvious that the intent is to use the same tolerance independent of units.